packages/ui-extensions/src/surfaces/admin/extension-points.ts

```diff
@@ -1,10 +1,39 @@
 import type {RenderExtension} from '../../extension';
 
-import type {AnyComponent} from './shared';
-import type {StandardApi} from './api';
+import type {AnyComponent, Components} from './shared';
+import type {StandardApi, CustomerSegmentationTemplateApi} from './api';
+import {AnyComponentBuilder} from '../../shared';
+
+type CustomerSegmentationTemplateComponent = AnyComponentBuilder<
+  Pick<Components, 'CustomerSegmentationTemplate'>
+>;
 
 export interface ExtensionPoints {
   Playground: RenderExtension<StandardApi<'Playground'>, AnyComponent>;
+  'Admin::Customers::SegmentationTemplatesFirstTimeBuyers::Show': RenderExtension<
```

Comment:
Are we going to wait until the conventions for extension point names is sorted out? `Show` is a term that they are using in that proposal currently, but it is different from checkout and I am not sure we have alignment on that part of the naming proposal.